vsts/pipelines/templates/_buildimageBasesJobTemplate.yml

```diff
@@ -25,6 +25,8 @@ jobs:
     inputs:
       scriptPath: ${{ parameters.scriptPath }}
       args: ${{ parameters.imageDir }} ${{ parameters.imageDebianFlavor }}
+    env:
+      DOTNET_PRIVATE_STORAGE_ACCOUNT_ACCESS_TOKEN: $(DotnetPrivateStorageAccountAccessToken)
```

Comment:
Hope I didn't miss something: where is this one used? Was it something introduced in the previous PR?

Reply:
@daniv-msft Yes it was! This is the environment variable that will be inserted into the staging base Dockerfile as a secret, at this line https://github.com/microsoft/Oryx/blob/main/build/buildRunTimeImageBases.sh#L156

Reply:
Thanks!